tests/unit/lfp/lfp.cpp

```diff
@@ -32,7 +32,7 @@ BOOST_AUTO_TEST_CASE(LFP_PointSource_LineSource) {
     std::array<double, 3> segment_end =
         paxpy(segment_start, 1.0, std::array<double, 3>{0.0, 0.0, segment_length});
     double floor{1.0e-6};
-    double pi = 3.141592653589;
+    pi = 3.141592653589;
```

Comment:
I do not understand: why is there still this hard-coded value?

Reply:
This is still hardcoded in the unit test because for this unit test we don't run `NEURON` to get it from there